parser_benchmark/benches/min-size.rs

```diff
@@ -0,0 +1,50 @@
+use std::env::temp_dir;
```

Comment:
Can this just be in `askama_parser/benches` instead?

Reply:
Done.